recipes/gym-notices/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "gym-notices" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gym-notices-{{ version }}.tar.gz
+  sha256: ad25e200487cafa369728625fe064e88ada1346618526102659b4640f2b4b911
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools >=42
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - gym_notices
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/Farama-Foundation/gym-notices
+  summary: Notices for gym
+  license: MIT
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - MattChanTK
```

Comment:
Not @thewchan?

Reply:
Ugh it SHOULD be but somehow my git client always put this in instead and sometimes I forget to change it...